sem_covid/entrypoints/notebooks/EDA/eda_wrangling/data_observations.py

```diff
@@ -21,24 +21,27 @@ def plot_pie_chart(observations: pd.DataFrame, chart_title: str) -> px:
     """
     columns = observations.columns
 
-    return px.pie(observations, values=columns[1], names=columns[0], title=chart_title)
+    return px.pie(observations, values=columns[1], names=columns[0], title=chart_title, opacity=0.8,
+                  color_discrete_sequence=['#003d66', '#004d80', '#006bb3', '#008ae6', '#1aa3ff', '#4db8ff', '#80ccff',
```

Comment:
oh! This approach looks tedious.  I thought there is an automatic way to define a palette, or in fact using a predefined palette; but fine for now